vNext/AISKU/README.md

```diff
@@ -189,9 +185,32 @@ appInsights.downloadAndSetup({
 	url: "https://az416426.vo.msecnd.net/beta/ai.2.min.js"
 });
 ```
-
 Test in internal environment to verify monitoring telemetry is working as expected. If all works, please update your api signatures appropriately to SDK V2 version and deploy in your production environments.
 
+## Build a new extension for the SDK:
+The beta SDK supports the ability to include multiple extensions at runtime. In order to create a new extension, please implement the following interface:
+
+	> ITelemetryPlugin
```

Comment:
Include link to file here instead of new line? Or paste the code-formatted interface here. The `>` is being rendered